src/migrations/migration.js

```diff
@@ -0,0 +1,40 @@
+const createUser = `
+    DROP TABLE IF EXISTS users;
+    CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
```

Comment:
what does this do?

Reply:
_uuid-ossp_ is a contribution module and it is not loaded in the server by default, I loaded it in the server so I can use UUID in the id column